wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WirePlugin.kt

```diff
@@ -235,8 +233,8 @@ class WirePlugin : Plugin<Project> {
         if (extension.protoLibrary) {
           task.protoLibraryOutput.set(File(project.libraryProtoOutputPath()))
         }
-        task.sourceInput.set(protoSourceProtoRootSets.inputLocations)
-        task.protoInput.set(protoPathProtoRootSets.inputLocations)
+        task.sourceInput.set(project.provider { protoSourceProtoRootSets.inputLocations })
+        task.protoInput.set(project.provider { protoPathProtoRootSets.inputLocations })
```

Comment:
To make this lazier we need to refer to the configurations symbolically. I don’t know how to do that.

Reply:
👍